.env.example

```
@@ -9,3 +9,9 @@ export API_KEY_POLYGONSCAN="YOUR_API_KEY_POLYGONSCAN"
 export API_KEY_SNOWTRACE="YOUR_API_KEY_SNOWTRACE"
 export MNEMONIC="YOUR_MNEMONIC"
 export FOUNDRY_PROFILE="default"
+export FEE=0X0
+export VALUE=0
+export NONCE=0
+export CALLDATA="YOUR_CALL_DATA"
+export FEE_RECIPIENT=0x0
```

```suggestion
export MSG_CLAIM_FEE=0X0
export MSG_CLAIM_VALUE=0
export MSG_CLAIM_NONCE=0
export MSG_CLAIM_CALLDATA="YOUR_CALL_DATA"
export MSG_CLAIM_FEE_RECIPIENT=0x0
```
@od-hunter Could you please prefix the values with smth like "MSG_CLAIM" if you don't mind? I thought it would better separate the vars logically